src/app/components/bibliography/bibliography-item/bibliography-item.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+import { BibliographicCitation } from 'src/app/services/xml-parsers/bibliography-parser.service';
+
+@Component({
+  selector: 'evt-bibliography-item',
+  templateUrl: './bibliography-item.component.html',
+  styleUrls: ['./bibliography-item.component.scss'],
+})
+export class BibliographyItemComponent {
+  @Input() biblField: BibliographicCitation;
+  @Input() authors: string;
+  @Input() titles: string;
+  @Input() editors: string;
+  @Input() pubPlaces: string;
+  @Input() publishers: string;
+  @Input() dates: string;
```

Comment:
Why dates are strings?

Reply:
In TEI dates can also ben encoded with missing information that cannot lead to a precise Date (e.g. `<date when="1100--">`), that's true, but in the case of bibliographic reference it is very difficult to have an incomplete date. Thus I agree with @szenzaro that this property shod not be a simple `string`.